packages/atoms/src/Select.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { FlagIconName, Icon } from '@trezor/icons';
+
+import { Box } from './Box';
+import { Text } from './Text';
+import { BottomModal } from './Modal/BottomModal';
+import { SelectItem, SelectItemProps } from './SelectItem';
+
+export type SelectItem = {
+    label: string;
+    value: string;
+    iconName: FlagIconName;
```

Comment:
I believe icon will be optional

Reply:
[fc5788e](https://github.com/trezor/trezor-suite/pull/5506/commits/fc5788e4a643dcc240d755543b1f56d03fefbea4)

Reply:
I added it as optional